lib/team.rb

```diff
@@ -3,13 +3,15 @@ class Team
               :franchiseid, 
               :teamname, 
               :abbreviation, 
-              :stadium
+              :stadium,
```

Comment:
do we need stadiums in any other method? should it just be removed as an attribute altogether?

Reply:
I can go and check that everything still runs with it's removal and push those changes form a new branch!